website/layouts/docs.erb

```diff
@@ -3,535 +3,522 @@
-            <a href="/docs/modules/composition.html">Module Composition</a>
+          <li>
+            <a href="/docs/commands/taint.html"><code>taint</code></a>
           </li>
-        </ul>
-      </li>
 
-      <li<%= sidebar_current("docs-backends") %>>
-        <a href="/docs/backends/index.html">Backends</a>
-        <ul class="nav">
-          <li<%= sidebar_current("docs-backends-config") %>>
-            <a href="/docs/backends/config.html">Configuration</a>
+          <li>
+            <a href="/docs/commands/validate.html"><code>validate</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-backends-init") %>>
-            <a href="/docs/backends/init.html">Init</a>
+          <li>
+            <a href="/docs/commands/untaint.html"><code>untaint</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-backends-state") %>>
-            <a href="/docs/backends/state.html">State Storage & Locking</a>
+          <li>
+            <a href="/docs/commands/workspace/list.html"><code>workspace list</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-backends-operations") %>>
-            <a href="/docs/backends/operations.html">Remote Operations</a>
+          <li>
+            <a href="/docs/commands/workspace/select.html"><code>workspace select</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-backends-types") %>>
-            <a href="/docs/backends/types/index.html">Backend Types</a>
+          <li>
+            <a href="/docs/commands/workspace/new.html"><code>workspace new</code></a>
           </li>
-        </ul>
-      </li>
 
-      <li<%= sidebar_current("docs-plugins") %>>
-        <a href="/docs/plugins/index.html">Plugins</a>
-        <ul class="nav">
-          <li<%= sidebar_current("docs-plugins-basics") %>>
-            <a href="/docs/plugins/basics.html">Basics</a>
+          <li>
+            <a href="/docs/commands/workspace/delete.html"><code>workspace delete</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-plugins-signing") %>>
-            <a href="/docs/plugins/signing.html">Signing</a>
+          <li>
+            <a href="/docs/commands/workspace/show.html"><code>workspace show</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-plugins-provider") %>>
-            <a href="/docs/plugins/provider.html">Provider</a>
+          <li>
+            <a href="/docs/commands/0.12upgrade.html"><code>0.12upgrade</code></a>
           </li>
 
-          <li<%= sidebar_current("docs-internals-plugins") %>>
-            <a href="/docs/internals/internal-plugins.html">Internals</a>
+          <li>
+            <a href="/docs/commands/0.13upgrade.html"><code>0.13upgrade</code></a>
           </li>
         </ul>
       </li>
 
-      <li<%= sidebar_current("docs-internals") %>>
-        <a href="/docs/internals/index.html">Internals</a>
+      <li>
+        <a href="#">Internals</a>
```

Comment:
You found one of the things I went back and forth on multiple times. 😑 There was a period where I was 100% sure this should move to Extend, but then _something_ changed my mind -- it might have been the provider network mirror protocol page, but now I'm finding that unconvincing. 

What do you think, move to Extend? Or somewhere else?

Reply:
FWIW I've not really been a fan of this "Internals" grab-bag in general -- it seems to have mostly just become a place for things we couldn't find another home for -- so I'd be totally on board with splitting it up into some other sections elsewhere, though I will admit I don't have a great home to suggest for most of them.

For the "Terraform-native service protocols" though, I could imagine them having their own subsection somewhere ("Extend" does seem like a reasonable container) which leads with the general info we currently have on the Service Discovery Protocol page and then has each of the specific protocols as sub-pages. (The discovery page is already organized a bit like a landing page for protocols, but its current placement in the navigation makes that a little awkward.)